build-system/tsconfig.json

```diff
@@ -26,7 +26,16 @@
     "./common/**/*.js",
     "./common/**/*.ts",
     "./server/**/*.js",
-    "./server/**/*.ts"
+    "./server/**/*.ts",
+    "./tasks/**/*.ts",
+    "./tasks/**/*.ts"
   ],
-  "exclude": ["**/node_modules", "common/check-package-manager.js"]
+  "exclude": [
+    "**/node_modules",
+    "common/check-package-manager.js",
+    "server/app-index/test/test-html.js",
+    "server/app-index/test/test-amphtml-helpers.js",
+    "server/new-server/transforms/dist",
```

Comment:
Maintenance nit: `**/node_modules` and `server/new-server/transforms/dist` are output directories that are not part of the codebase. Would it be possible to just ignore them while running checks instead of having to explicitly list them here? Would the change in #32960 help with this?